proto/sentry_protos/snuba/v1/endpoint_aggregate_bucket.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message AggregateBucketRequest {
+  RequestMeta meta = 1;
+  enum Function {
+    FUNCTION_UNSPECIFIED = 0;
+    FUNCTION_SUM = 1;
+    FUNCTION_AVERAGE = 2;
+    FUNCTION_COUNT = 3;
+    FUNCTION_P50 = 4;
+    FUNCTION_P95 = 5;
+    FUNCTION_P99 = 6;
+    FUNCTION_AVG = 7;
+  }
+
+  Function aggregate = 4;
+  TraceItemFilter filter = 5;
+  uint64 granularity_secs = 6;
+  AttributeKey key = 7;
+  repeated VirtualColumnContext virtual_column_context = 8;
```

Comment:
Not sure how this will be used here yet. Is it meant for the top n use case?

Reply:
I'm happy to remove it for now